pyscitt/pyscitt/local_key_sign_client.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.serialization import load_pem_private_key
+from loguru import logger as LOG
+
+from pyscitt.client import MemberAuthenticationMethod
+
+from . import crypto
+
+
+class LocalKeySignClient(MemberAuthenticationMethod):
+    def __init__(self, cert: str, key: str) -> None:
+        self.cert = cert
+        self.key = key
+        self.key_id = crypto.get_cert_fingerprint(cert)
+        self.private_key = load_pem_private_key(
+            key.encode("ascii"),
+            password=None,
+            backend=default_backend(),
+        )
```

Comment:
```suggestion
        self.private_key = load_pem_private_key(
            key.encode("ascii"),
            password=None,
        )
```

See https://github.com/microsoft/scitt-ccf-ledger/pull/62